docs/content/reference/data_types.md

```diff
@@ -39,5 +39,4 @@ generic container for arrays of data. Images are restricted to tensors of rank 2
 
 ## Other **Archetypes**
 * [Scalar](data_types/scalar.md): a single scalar / metric value. Can be viewed in the `TimeSeries` space view.
-* [TextEntry](data_types/text_entry.md): captures text data. Can be viewed in the `Text` space view.
```

Comment:
bit unfortunate to throw it away entirely and thus any mention of the ability to log text.
But I suppose putting up the new doc pages with all the examples and generated images is orthogonal